terraform/app/ssm_parameters.tf

```
@@ -1,38 +1,7 @@
-resource "aws_ssm_parameter" "good_job_max_threads" {
-  name = "/${var.environment}/good_job_max_threads"
-  type = "String"
+resource "aws_ssm_parameter" "environment_config" {
```

```suggestion
resource "aws_ssm_parameter" "environment_configs" {
```

Not sure what the convention is with Terraform, but when using `fore_each` would we normally pluralise the resource name?

---

But if there is a strong preference/existing trend in the repository for this behavior I am also not so fussed